npm/binary-install.js

```diff
@@ -0,0 +1,114 @@
+const { existsSync, mkdirSync, createWriteStream } = require("fs");
+const { join } = require("path");
+const { spawnSync } = require("child_process");
+
+const axios = require("axios");
+const rimraf = require("rimraf");
```

Comment:
I guess we can use nodejs builtin functions of fs for removing files.

Reply:
hmm will do